### PR TITLE
feat: changed style to be inline with other services

### LINKED ISSUE #1157: Change values.yaml style for mqttbridge, sensorconnect & cameraconnect to be in line with the others
**Is your feature request related to a problem? Please describe.**
While developing AutoDeploy this issue prevents me from deploying the correct version for these pods

**Describe the solution you'd like**
Change values.yaml to have 
```
image:
  repository: abc
```
for these pods

### PR DESCRIPTION
**Pull request by Scarjit**
# Description

This pr changes the style of the values.yaml file for mqttbridge, sensorconnect, cameraconnect to match the style of the other services

Fixes #1157 

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] I have checked that my issue complies with the [Contributing guidelines](https://github.com/united-manufacturing-hub/united-manufacturing-hub/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: deployment/united-manufacturing-hub/values.yaml [hunk 1]
```diff
@@ -979,7 +979,8 @@ kafkastatedetector:
 ### mqttbridge ###
 mqttbridge:
```
Comment (JeremyTheocharis) [inline deployment/united-manufacturing-hub/values.yaml:980]: THis requires again an tutorial on upgrading

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

**Review by JeremyTheocharis — COMMENTED** [review]
Are you sure this is not a breaking change? Cant they not any backwards compatibility issues?

**Comment by JeremyTheocharis** [comment]
> Are you sure this is not a breaking change? Cant they not any backwards compatibility issues?

@Scarjit Reminder that this is still open

**Comment by Scarjit** [comment]
@JeremyTheocharis Unsure if this is breaking.

**Pull request closed without merging**